packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -10,11 +10,17 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   const legacy =
     typeof pluginOptions.legacy !== `undefined` ? pluginOptions.legacy : true
 
-  // If icons were generated, also add a favicon link.
+  // The user has an option to opt out of the favicon link tag being inserted into the head.
   if (pluginOptions.icon) {
     let favicon = icons && icons.length ? icons[0].src : null
 
-    if (favicon) {
+    let insertFaviconLinkTag = Object.keys(pluginOptions).includes(
```

Comment:
if you look at line 10 we check if a property exists a little bit easier. The way you do it looks a bit more work.
```suggestion
    const insertFaviconLinkTag = typeof pluginOptions.include_favicon !== `undefined` ? pluginOptions.include_favicon : true
```